gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,26 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<h1>We can't find this page</h1>
```

Comment:
Can you put the h1/lead p in a page header component from the design system as per other pages please?

Reply:
Should it also be a grey panel as with the rest of NICE? Like so?
![image](https://user-images.githubusercontent.com/19930229/88189733-44ebf680-cc31-11ea-91a0-dd4eddef78d1.png)